app/javascript/utilities/hooks/useGlobalKeyEventListener.js

```diff
@@ -0,0 +1,32 @@
+import { useEffect } from 'preact/hooks';
+
+export const registerGlobalKeyEventListener = (keys, callback) => {
```

Comment:
This is exported to support the use of global key event listener on class-based components